src/main/steps/applicant1/connection-summary/content.ts

```diff
@@ -5,28 +5,47 @@ import { CommonContent } from '../../common/common.content';
 
 export const jurisdictionMoreDetailsContent = (
   connections: JurisdictionConnections[] | undefined,
+  isDivorce: boolean,
   isRespondent = false
 ): { connectedToEnglandWales: string; readMore: string } => {
-  const resConnection = enContainsHabitualResConnection(connections);
+  const habConnection = enContainsHabitualResConnection(connections);
   const domConnection = enContainsDomConnection(connections);
+  const residualConnection = enContainsResidualConnection(connections, isDivorce);
 
-  const connectionIndex = isRespondent || (resConnection && domConnection) ? 2 : domConnection ? 1 : 0;
+  const connectionIndex =
+    isRespondent || (habConnection && domConnection && residualConnection)
+      ? 3
+      : domConnection
```

Comment:
this logic doesnt seem to consider what it was before, did something change? before hab and dom were true, its 2 but here if its true itll automatically go to 1 (considering residual is false).